### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by jadhu22** [fangchangma/self-supervised-depth-completion#11]
from the kiti dataset, how is the data to be distributed in the data folder, with kitti_depth and kitti_rgb as its sub-folders?? I want to test he pre-trained model only. Please tell me what data do I need

**Comment by fangchangma** [comment]
Hi. Hopefully the data directory structure is clear from the README file (copied and pasted here):
```
.
├── self-supervised-depth-completion
├── data
|   ├── kitti_depth
|   |   ├── train
|   |   ├── val_selection_cropped
|   └── kitti_rgb
|   |   ├── train
|   |   ├── val_selection_cropped
├── results
```

**Comment by jadhu22** [comment]
Yes, but in the link provided for kitti dataset, there is link for four files + raw data(images). For testing, is the "selection data" file to be used? I want to know how have you separated the whole data into train and val_cropped_selected. Please let me know. Thanks

**Comment by gautamjain1009** [comment]
Hi!, i find your work really interesting. I want to evaluate your architecture with pre-trained weights only. what should be the structure of the data folder in that case.

The thing is kitti_rgb train is so big and i have to transfer all the files via SSH.

**Comment by fangchangma** [comment]
> what should be the structure of the data folder in that case

The data folder structure is illustrated in the [README](https://github.com/fangchangma/self-supervised-depth-completion#requirements). You can switch to any data loader, as long as the final loaded data format is the same

**Comment by gautamjain1009** [comment]
@jadhu22 
 Are you able to run with the pretrained models?  If, yes which parameters you passed in the argument while calling the main.py or did you change something in the code.
